src/og/control/LayerAnimation.js

```diff
@@ -17,30 +17,25 @@ class LayerAnimation extends Control {
     constructor(options = {}) {
         super(options);
 
-        this.events = new Events(["change"])
+        this.events = new Events(["change", "idle", "play", "pause", "stop"])
 
-        this._name = `layerAnimation ${this._id}`;
-
-        this._layersArr = options.layers || [];
-        this._layersIndexesArr = this._layersArr.map((l) => l._id);
+        this._name = `layerAnimation-${this._id}`;
```

Comment:
why need this._id in name, simply layerAnimation?

Reply:
For multiple LayerAnimation controls.